peggy_on_the_hub.md

```diff
@@ -0,0 +1,45 @@
+# Peggy
+
+## Question
+
+Should Peggy be deployed as a Shared Security Zone or module in Gaia as part of the Cosmos SDK?
+
+## Context
+
+In April 2020, Althea proposed an [DeFi enabled Ethereum bridge for Cosmos](https://blog.althea.net/defi-enabled-ethereum-bridge-for-cosmos/) for a DEFI enable PegZone.
+
+The Interchain Foundation has provided [Althea](https://blog.althea.net/solid-foundations-for-peggy/)with resources to implement the technical components of this vision.
```

Comment:
I think an argument could be made that because ATOM holders have paid for Peggy dev that it should be deployed on the hub.

Reply:
I agree. Most growth potential of peggy should be redistributed to Atom holders, except for some fraction to the operational institution who build, manage and operate peggy backend/frontend&interfaces/community/governance/branding&marketing. The fraction can be the institution's payoff over operating cost and opportunity cost.